test/controllers/add.trust.controller.spec.ts

```diff
@@ -91,6 +103,46 @@ describe("Add Trust Controller Tests", () => {
     });
   });
 
+  describe("GET tests with url params", () => {
+    test(`renders the ${config.ADD_TRUST_PAGE} page`, () => {
+      mockGetUrlWithParamsToPath.mockReturnValueOnce(MOCKED_URL);
+      (getApplicationData as jest.Mock).mockReturnValue(mockAppData);
+
+      mockIsActiveFeature.mockReturnValueOnce(true);
```

Comment:
Worth adding a comment to document which flag we are mocking.

Reply:
Done